### ISSUE TITLE
[PSA] Airbnb iOS has migrated to Bazel

### ISSUE THREAD

**Issue by qyang-nj** [airbnb/BuckSample#161]
Airbnb iOS has migrated the build system from Buck to [Bazel](https://bazel.build/). Since we will no longer use Buck internally, we are going to archive this BuckSample repo.

**Comment by winstondu** [comment]
@qyang-nj , will there be a Bazel sample?

Also, what is the impetus behind moving to Bazel at Airbnb?

**Comment by qyang-nj** [comment]
@winstondu There are plenty of Bazel examples available, such as those in [bazelbuild/rules_apple](https://github.com/bazelbuild/rules_apple/tree/master/examples).

We detailed the reasons behind and the process of our migration from Buck to Bazel in [this blog post](https://medium.com/airbnb-engineering/migrating-our-ios-build-system-from-buck-to-bazel-ddd6f3f25aa3). Hopefully, you find it helpful.

**Comment by winstondu** [comment]
@qyang-nj I'm curious though--
How does building via bazel support intelligent IDE features like goto definition or intellisense?

Do you still use XCode's symbol indexing to support your iOS developers?

**Comment by qyang-nj** [comment]
@winstondu Yes, of course we support indexing in Xcode. There are a few approaches to support it. If you're new to Bazel on iOS, I recommend taking a look at https://github.com/MobileNativeFoundation/rules_xcodeproj

**Comment by winstondu** [comment]
I think my question is predicated on whether some of your iOS Engineers use something other than xcode as the IDE.

For example, if some are using VSCode, how are you ensuring Bazel communicates with sourcekit-lsp?
